kvcached/integration/sglang/interfaces.py

```diff
@@ -15,38 +15,42 @@
 _kvcached_initialized: bool = False
 _kvcached_device = None
 _async_sched = False
+_contiguous_layout = True
 
 
 def init_kvcached(tp_rank: int = 0,
                   tp_size: int = 1,
                   device: Optional[str] = None,
-                  async_sched: bool = False) -> None:
-    global _kvcached_initialized, _kvcached_device, _async_sched
+                  async_sched: bool = False,
+                  contiguous_layout: bool = True) -> None:
```

Comment:
I kinda feel that the contiguous_layout is a kvcached-specific config. Does the engine needs to setup this flag? Or if enabling this, does the engine needs to do additional things?